### DISCUSSION TITLE
Documentation translations

### DISCUSSION THREAD

**Discussion by HZooly** [nuxt/nuxt#16054] [Ideas]
Do you plan to translate Nuxt 3 documentation as it is done for v2?

**Comment by pi0** [comment]
Hi @HZooly! Of course but the translations process will be possibly outsourced to the main docs repo. This way we can make sure documentation keeps up to date. Also, we also started using auto-generated docs for configuration [directory-structure](https://v3.nuxtjs.org/docs/directory-structure/nuxt) is directly extracted from schema. Translation might be tricky for them.

**Reply by nazarepiedady** [reply]
@pi0 about the **directory-structure** we can continue to generate it automatically for the
English documentation and to process it manually for the others languages.

**Comment by kissu** [comment]
If some human help is needed, people will be okay with helping (myself included)! ✋🏼

**Comment by AdamModus** [comment]
Same here! I'd like to help with translations as well! 😊

**Comment by pi0** [comment]
@kissu @AdamModus Invitations sent for new translations project.

Everyone if love to dedicate some time helping to translate nuxt.js documentation, please comment with a short introduction and interested language(s) 💚

**Reply by Ayred** [reply]
Hello, I am Ayred, I would love to help with portuguese. See you @pi0

**Reply by nazarepiedady** [reply]
@pi0 could you send me an invitation for the new translations project?

**Reply by ojvribeiro** [reply]
I think we should create a pt and a pt-br version.

**Comment by nozomuikuta** [comment]
@pi0 

If you need (more?) people for Japanese translation, my first language, I'm grad to help. 🙋‍♀️ 

I'm also motivated to translate Nuxt modules' docs as well, 
if you think that enough number of people is already working on the main documentation.

**Reply by pi0** [reply]
Sent invitation.

**Comment by Rainer-Yu** [comment]
@pi0
If you need (more) people to translate Chinese, I am happy to help.

**Reply by chaos-zhu** [reply]
me too~

**Reply by evanlong-me** [reply]
me too~ + 1

**Reply by ModyQyW** [reply]
me too~

**Reply by codingcn** [reply]
Chinese mirror site.    https://nuxt.com.cn/


see https://github.com/nuxt/nuxt/discussions/16208

**Comment by sykes10** [comment]
I can help with translations to spanish

**Reply by Bernardao** [reply]
I can help too with Spanish translation if needed

**Comment by r9mp** [comment]
I can help with translations to french if help is still needed.

**Reply by edimitchel** [reply]
Vue 3 french translation leader here, I'm also interested to help. @pi0

**Comment by vylenne** [comment]
I can help with russian, too!

**Reply by mihail727** [reply]
\+

**Reply by AndreyYolkin** [reply]
+1, I'm pleased to help

**Reply by Ibochkarev** [reply]
@vylenne @mihail727  https://github.com/nuxt/nuxt/discussions/16054#discussioncomment-9635452

**Comment by maxdzin** [comment]
@pi0 
If you need help with the translation to Ukrainian, I'm happy to help :blue_heart::yellow_heart:

**Reply by avinean** [reply]
`+`

**Reply by logotip4ik** [reply]
`+`

**Comment by nestle49** [comment]
i can help with ukrainian, russian

**Reply by bot08** [reply]
+

**Comment by nazarepiedady** [comment]
I am here to translate the entire website and documentation to Portuguese as I did to the second version of the framework.

**Comment by lucasdavidferrero** [comment]
I'm willing to help with Spanish, glad to help.

**Comment by necatikcl** [comment]
I can help with Turkish translations if it is possible. Turkish is my primitive language and I am a college student of English.

**Comment by Coder-Iro** [comment]
I think I can help translate into Korean

**Comment by hamzaelmaghari** [comment]
Who wants translation in Arabic! I can help .

**Comment by Zielgestalt** [comment]
I can help with German

**Comment by Semvrij** [comment]
I can help with the translation of the documentation into Dutch 🇳🇱

**Comment by lucavenir** [comment]
I can help with italian translations if they are needed 🇮🇹

**Comment by YusufcanY** [comment]
I can help for the Turkish!

**Comment by Rachiid007** [comment]
I can help to translate into French.

**Comment by gacek1123** [comment]
I could help with Polish translations, it's my native language. I already have some experience with translating too.

**Comment by Joaojack** [comment]
Hi, I can contribute to the french translation as well.

**Comment by desigrammer** [comment]
Hello, I would like to help translate into Japanese, thank you.

**Comment by chz** [comment]
HI, I can help translate and contribute to Azerbaijani

**Comment by alessioacella** [comment]
Hi there, I can contribute to the Italian translation as well.

**Comment by Gruce** [comment]
Hello @pi0 ,
I can help with the Arabic language.

**Comment by KingSit3** [comment]
hello, i can contribute to Indonesian Language 😁

**Comment by ShellarFX** [comment]
Hello! I can help with russian translation. I have been studying English for 11 years

**Comment by Vinccool96** [comment]
I too can help with French translation, especially if Canadian French is a thing

**Reply by kissu** [reply]
I guess we will stick to the regular one and keep it as only 1 🇫🇷. Otherwise, Catalan, Basque etc etc...will be quite too complex and not that relevant. The goal is to help with overall language IMO.

**Reply by Vinccool96** [reply]
I can still help with French.

**Comment by younho9** [comment]
Hello! I think I can help translate into Korean.

**Comment by nazarepiedady** [comment]
When can we start to translate the documentation @pi0 ?

And could you @pi0  send me the invitation to the translation project?

**Comment by Chuiantw1212** [comment]
@pi0 
Hi, I'm Ian. I built and deployed a job searching website using Nuxt3 and Fastify.js. 
Basically all my career is about handling Vue based projects. 
The repo is made public here (https://github.com/Chuiantw1212/job-pair-nuxt3)

May I help translate document into Chinese?

**Comment by Amirali-Yavari** [comment]
Hi dear i can help you in translation to persian

@pi0

**Comment by nazarepiedady** [comment]
Any news about this topic?

**Comment by tell-k** [comment]
Is this translation plan still alive?
If it is still alive, I'd like to contribute in Japanese.

**Reply by nazarepiedady** [reply]
@tell-k I recommend you talk to @danielroe for more details about this thread, he is too responsive.

**Comment by amiria703** [comment]
@pi0 @danielroe 
Any news on this? I'd like to translate whole website and docs into Persian?

**Comment by savionascimentodev** [comment]
Any news about this topic?

**Reply by nazarepiedady** [reply]
@savionascimentodev, as I am the person behind the officially served Portuguese documentation of Nuxt 2, I am organising the bases for us to start the translation work, I will warn when things be ready.

**Reply by savionascimentodev** [reply]
Wow, you will also translate nuxt 3, count on me please 😀.

**Comment by AloisSeckar** [comment]
As I recently managed to get [Czech translation of Vue.js docs](https://cs.vuejs.org/) into official WIP state, I am offering translating Nuxt docs into Czech as well, once it is possible :)

**Comment by Mat4m0** [comment]
@danielroe @pi0 
Any news on this topic?

**Comment by taku-y-9308** [comment]
Are there still projects to be translated into Japanese?
If so, I would like to participate!

**Comment by shayr1** [comment]
I would love to help with translations of the documentations if still needed - more people need to changes sides to the best Vue side :)

**Comment by EvanSchleret** [comment]
Don't know if it still necessary but I can help to translate in French 🚀

**Comment by Ibochkarev** [comment]
Any news about this topic?

**Reply by AndreyYolkin** [reply]
Since nuxt [opened the sources](https://github.com/nuxt/nuxt.com/tree/main?tab=readme-ov-file#setup), we can attach another docs folder and try to run. Another translation-gang repo?)

**Comment by Ibochkarev** [comment]
Hi everyone!

Who has been waiting for Russian-language documentation on NUXT for a long time can join the translation of documentation into Russian!

https://nuxt-ru.vercel.app/ — the site is deployed here

Repositories for translation:
— site: https://github.com/translation-gang/nuxt.com
— documentation: https://github.com/translation-gang/nuxt/tree/main/docs

We are waiting for your PRs.

The team that translated https://ru.vuejs.org/ is already on board!

**Reply by nazarepiedady** [reply]
@HZooly, thanks for your contribution in the name of the community.

**Comment by 2nofa11** [comment]
@manniL 

I was the one who approached you at VueFes Japan 2024 about contributing to Nuxt!
I would love to help with the translation of Nuxt.
Please let me assist you with the Nuxt translation work!

**Comment by ghost** [comment]
Hi good people, I'd be happy to help with translation to Polish

**Comment by OrbisK** [comment]
I think we should track this here: https://github.com/nuxt/nuxt.com/issues/1711
